src/fmu/tools/qcforward/_qcforward.py

```diff
@@ -252,5 +260,12 @@ def actions_validator(actionsin: dict) -> dict:
             raise ValueError(
                 "Seems wrong to mix 'all' and 'any' in one line; be consistent"
             )
+        if "any" in str(vals):
+            anyhit = True
+        if "all" in str(vals):
+            allhit = True
+
+    if not anyhit or not allhit:
+        raise ValueError("One of 'any' or 'all' is missing in the actions input")
```

Comment:
```suggestion
        if  ("any" not in str(vals)) or ("all" not in str(vals)):
            raise ValueError("One of 'any' or 'all' is missing in the actions input")
```
This Error check should be within the loop, otherwise only the last action will be checked. 
Another suggestion, you could check for "any" and "all" directly here, and drop the 6 lines related to the `anyhit` and `allhit` variables